tests/drivers/motor_driver/main.c

```diff
@@ -74,7 +78,7 @@ void motion_control(void)
     xtimer_ticks32_t last_wakeup /*, start*/;
```

Comment:
```suggestion
    xtimer_ticks32_t last_wakeup;
```

I guess this is a dev leftover that was commented out? If not then simply this at least:

```suggestion
    xtimer_ticks32_t last_wakeup; /* start */
```

Reply:
Not sure what it is, it was added when the motor_driver driver was added 🤷‍♂️